src/pkg/notification/hook/hook.go

```diff
@@ -43,44 +39,42 @@ func (hm *DefaultManager) StartHook(ctx context.Context, event *model.HookEvent,
 		return err
 	}
 
-	t := time.Now()
-	id, err := hm.jobMgr.Create(ctx, &job_model.Job{
-		PolicyID:     event.PolicyID,
-		EventType:    event.EventType,
-		NotifyType:   event.Target.Type,
-		Status:       cModels.JobPending,
-		CreationTime: t,
-		UpdateTime:   t,
-		JobDetail:    string(payload),
-	})
-	if err != nil {
-		return fmt.Errorf("failed to create the job record for notification based on policy %d: %v", event.PolicyID, err)
+	extraAttrs := make(map[string]interface{})
+	if err = json.Unmarshal(payload, &extraAttrs); err != nil {
+		return err
 	}
-	statusHookURL := fmt.Sprintf("%s/service/notifications/jobs/webhook/%d", config.InternalCoreURL(), id)
-	data.StatusHook = statusHookURL
 
-	log.Debugf("created a notification job %d for the policy %d", id, event.PolicyID)
+	var vendorType string
+	switch event.Target.Type {
+	case model.NotifyTypeHTTP:
+		vendorType = job.WebhookJobVendorType
+	case model.NotifyTypeSlack:
+		vendorType = job.SlackJobVendorType
+	}
```

Comment:
default error?

Reply:
Checked the `vendorType` in the following section.